server/views/pages/prisons/incentiveGroups/addIncentiveGroup.njk

```diff
@@ -0,0 +1,95 @@
+{% extends "partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
+{% from "moj/components/page-header-actions/macro.njk" import mojPageHeaderActions %}
+{%- from "govuk/components/button/macro.njk" import govukButton -%}
+{%- from "govuk/components/input/macro.njk" import govukInput -%}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% set pageHeaderTitle = 'Add an incentive level group' %}
+{% set pageTitle = applicationName + " - Incentive groups - " + pageHeaderTitle %}
+
+{% set primaryNavItems = [
+  {
+    text: "Home",
+    href: "/"
+  },
+  {
+    text: "Prisons",
+    href: "/prisons",
+    active: true
+  }
+] %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ prison.name }}</span>
+        {{ pageHeaderTitle }}
+      </h1>
+
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      {% include "partials/errorSummary.njk" %}
+      {# Build incentive items array for checkboxes #}
+
+      {% set incentiveLevelItems = [] %}
+      {% for rangeKey, description in incentiveLevels %}
+        {% set incentiveLevelItems = (incentiveLevelItems.push({
+          value: rangeKey,
+          text: description
+        }), incentiveLevelItems)%}
+      {% endfor %}
+
+      <form action="/prisons/{{prison.code}}/incentive-groups/add" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+        {{ govukInput({
+          label: {
+            html: 'Incentive level group name',
+            classes: 'govuk-label--m'
+          },
+          id: 'name',
+          name: 'name',
+          classes: 'govuk-!-width-three-quarters govuk-!-margin-bottom-4',
+          value: formValues.name,
+          autocomplete: "off",
+          errorMessage: errors | findError('name')
+        }) }}
+
+        {{ govukCheckboxes({
+          name: "incentiveLevels",
+          fieldset: {
+            legend: {
+              text: "Which incentive levels is this group for?",
+              isPageHeading: true,
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          hint: {
+            text: "Select all that apply."
+          },
+          items: incentiveLevelItems,
+          values: formValues.incentiveLevels,
```

Comment:
Just tried this out locally and an an issue is that if you tamper with the checkbox values when submitting, these tampered values are returned when the page is reloaded. The user input (data) is whether or not the box is checked; **not** the value of the box.

See https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/c52ff110f19c5b51fa54443fa7a9b8b5e86600dc/server/views/pages/bookAVisit/additionalSupport.njk#L45 for an example of setting the `checked` value on each item rather than setting the `values` array. (n.b. fine in theory to use `values` array on this component but I think in the controller you'd want to sanitise it first to strip invalid values)